test-configs.yaml

```diff
@@ -600,6 +630,16 @@ device_types:
             - 'multi_v7_defconfig+kselftest'
           kernel: ['v3.18']
 
+  imx6q-sabresd:
+    mach: freescale
+    class: arm-dtb
+    boot_method: uboot
```

Comment:
Baseline results on staging: https://staging.kernelci.org/test/plan/id/5e9fa03dc0394ae66f61e03e/
All passing.

Upstream in LAVA:
https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/imx6q-sabresd.jinja2